cccat/main.py

```diff
@@ -126,6 +131,14 @@ async def root_view() -> RedirectResponse:
         event_consumer = init_event_consumer()
         event_consumer.run()
 
+    elif args[0] == "producer":
```

Comment:
Not sure what you try to do here, but publisher does not need separate thread because the API is the publisher/the one that emits events.